geonode_mapstore_client/client/js/routes/Home.jsx

```diff
@@ -294,9 +294,9 @@ function Home({
 
     const { query } = url.parse(location.search, true);
 
-    const queryFilters = Object.keys(query).reduce((acc, key) => key.indexOf('filter') === 0
-        ? [...acc, ...castArray(query[key]).map((value) => ({ key, value }))]
-        : acc, []);
+    const queryFilters = Object.keys(query).reduce((acc, key) => {
+        return [...acc, ...castArray(query[key]).map((value) => ({ key, value }))];
+    }, []);
```

Comment:
I think if we remove the control on the property key we will include also other query keys such as `sort`. The current params that represent the filters are:
- params that start with `filter`
- `f` for custom filters ids
- `q` for text search
- `extent` filter by extent

If we look in the this file queryFilters is used by:

- ConnectedDetailsPanel: maybe we can remove the prop filters from this component it does not seem to be used anymore

https://github.com/GeoNode/geonode-mapstore-client/blob/d2f74018cf4fe3195c53414dc840792e0960d85d/geonode_mapstore_client/client/js/routes/Home.jsx#L453

- FilterMenu: if queryFilters include all the possible filters we don't need to look for f, q or extent in filtersActive but instead use `queryFilters.length > 0`
https://github.com/GeoNode/geonode-mapstore-client/blob/d2f74018cf4fe3195c53414dc840792e0960d85d/geonode_mapstore_client/client/js/routes/Home.jsx#L495-L496

Reply:
right! I had forgotten the sort filter, it has not be added to queryFilters